hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.tsx

```diff
@@ -0,0 +1,358 @@
+            <Typography variant="body2" className={styles.heading}>
+                Datasheet
+            </Typography>
+            <Button
+                href={datasheet}
+                rel="noopener"
+                target="_blank"
+                startIcon={<LaunchIcon />}
+            >
+                Link
+            </Button>
+            <Typography variant="body2" className={styles.heading}>
+                Notes
+            </Typography>
+            {notes.split("\n").map((note, i) => (
+                <Typography key={i}>{note}</Typography>
+            ))}
+        </>
+    );
+};
+
+interface MainSectionProps {
+    name: string;
+    quantityAvailable: number;
+    quantityRemaining: number;
+    categories: string[];
+    picture: string;
+}
+const MainSection = ({
+    name,
+    quantityAvailable,
+    quantityRemaining,
+    categories,
+    picture,
+}: MainSectionProps) => {
+    const availability =
+        quantityRemaining === 0 ? (
+            <Typography color="secondary">OUT OF STOCK</Typography>
+        ) : (
+            <Typography className={styles.quantityAvailable}>
+                {quantityRemaining} OF {quantityAvailable} IN STOCK
+            </Typography>
+        );
+
+    return (
+        <div className={styles.mainSection}>
+            <div>
+                <Typography variant="h6">{name}</Typography>
+                {availability}
+                <Typography variant="body2" className={styles.heading}>
+                    Category
+                </Typography>
+                <div>
+                    {categories?.length > 0 &&
+                        categories.map((category, i) => (
+                            <Chip
+                                label={category}
+                                size="small"
+                                className={styles.categoryItem}
+                                key={i}
+                            />
+                        ))}
+                </div>
+            </div>
+            <img src={picture} alt={name} />
+        </div>
+    );
+};
+
+export const ProductOverview = ({
+    showAddToCartButton,
+}: {
+    showAddToCartButton: boolean;
+}) => {
+    let categoryNames: string[] = [];
+    let maxPerTeam: number | null = null;
+    let constraints: string[] = [];
+
+    const dispatch = useDispatch();
+    const closeProductOverview = () => dispatch(removeProductOverviewItem());
+
+    const isProductOverviewVisible: boolean = useSelector(
+        isProductOverviewVisibleSelector
+    );
+    const hardware = useSelector(hardwareBeingViewedSelector);
+    const categories = useSelector((state: RootState) =>
+        selectCategoriesByIds(state, hardware?.categories || [])
+    );
+    if (categories.length > 0) {
+        categoryNames = categories
+            .filter((category): category is Category => !!category)
+            .map((category) => category.name);
+        constraints = hardware?.max_per_team
```

Comment:
```suggestion
        constraints = hardware?.max_per_team !== undefined
```

Minor change, but it allows us to set max per team to 0 in the event we want to stop allowing an item to be checked out for some reason. No idea why we'd do that, but now we can!